content.properties

```
@@ -1,8 +1,26 @@
-#name=Ref 3.x distro
-#version=3.0.0
-#war.openmrs=2.6.1-SNAPSHOT
-omod.fhir2=1.9.0
-omod.webservices.rest=2.40.0-SNAPSHOT
-spa.frontendModules.@ohri/openmrs-esm-ohri-hiv-care-treatment-app=latest
-spa.frontendModules.@ohri/openmrs-esm-ohri-core-app=latest
-#spa.frontendModules.@openmrs/esm-primary-navigation-app=next
+# content.properties - Metadata for the content package and its dependencies
+
+# The name of this content package
+name=openmrs-content-hiv
```

```suggestion
name=hiv
```

---

Ideally, however, this would be populated from the POM directly:

```suggestion
name=${project.artifactId}
```

---

The above recommended change has been implemented, I don't know much about the data in content.properties, Amos will be responsible for the data. i just have name and version for the time being.